Lib/ast.py

```diff
@@ -562,15 +562,19 @@ def main():
     parser.add_argument('-m', '--mode', default='exec',
                         choices=('exec', 'single', 'eval', 'func_type'),
                         help='specify what kind of code must be parsed')
+    parser.add_argument('-t', '--type-comments', default='store_true',
```

Comment:
Since the default is `True` wouldn't it be better to have a `--no-type-comments` flag?

Reply:
Ah, looks like a typos. They probably meant:

```suggestion
    parser.add_argument('-t', '--type-comments', action='store_true',
```

Reply:
Agreed